accessibility/accessibility-checklist.md

```diff
@@ -39,6 +39,30 @@ Resources:
 * [Tool: HTML5 accessibility feature support for sections](http://www.html5accessibility.com/#sections)
 
 
+### Provide a Skip Navigation option
+
+Not all users are able to navigate by landmarks. For those users the multiple links typically found at the head of a document all have to be read via assistive technology, or laboriously tabbed through using a keyboard.
+
+To help them out you can add a "Skip Navigation" link to the top of each page which, when clicked, will take them to the main section of the page.
+
+We do this:
+
+```
+<body>
+<p><a href="#maincontent">Skip to main content</a></p>
+...
+<main id="maincontent">
```

Comment:
One thing that Link do is to add a `tabindex="-1"` to the `main`, which allows it to more consistently receive focus when using the skip link. I feel like we probably need to have that in the example (as not having it will cause bugs), but I'm wary of having both too many explanations, or of having much stuff in the example that isn't explained 🤔 

I wonder if skip links would be a good candidate for a component? Then we could link off to the component at a later date instead of explaining all the bits

Reply:
Good idea. Added a wee bit about that.

Reply:
And yeah, I think this is an excellent candidate for a component!